.circleci/config.yml

```diff
@@ -75,13 +81,14 @@ jobs:
 
   sam_build:
     docker:
-      - image: amazon/aws-sam-cli-build-image-python3.8:latest
+      - image: public.ecr.aws/sam/build-python3.12:1.124.0-20240911233354
```

Comment:
This seems like a really specific image to pin to.
On all the other projects where we are using the SAM images, we're targetting
`public.ecr.aws/sam/build-python3.12:latest`
any particular reason for this?

Reply:
No, this was the latest version I found, but can loosen `public.ecr.aws/sam/build-python3.12:latest`

Reply:
Fixed with https://github.com/DemocracyClub/electionleaflets/pull/255/commits/9e32842c4153b22a8ed38a4d335698cef83ed8c9.